extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -1294,21 +1304,21 @@ export class Bind {
 
   /**
    * Print out the current state in the console.
+   * @param {string=} opt_expression
```

Comment:
just noting that optional parameters need not be prefixed with opt anymore though I actually prefer it.

Reply:
AMP's (Malte's) style guide still uses them. `opt_` prefix is not necessary for params with default values though.